### PR TITLE
remove hardcoded DNS servers

### LINKED ISSUE #54: dns servers hardcoded in generic/ubuntu1804
The following servers are hardcoded in the generic/ubuntu1804 image:

```# systemd-resolve --status
Global
         DNS Servers: 4.2.2.1
                      4.2.2.2
                      208.67.220.220```

Wouldn't it be expected that the local DNS from DHCP should have priority over these servers? If running in an environment where access is blocked to external DNS servers, then this image requires further work to use, rather than working out of the box, like say, bento/ubuntu-18.04 or ubuntu/bionic64.

### PR DESCRIPTION
**Pull request by furlongm**
First pass at removing hardcoded DNS servers so these boxes can use the DHCP-supplied DNS server.

Closes #54

### CODE REVIEW

**Comment by ladar** [comment]
Leaving this open for now. I'll need to do some testing before merging. 

I assume these changes mean the boxes won't work if they don't get a DNS server via DHCP?

Any idea how we can setup fall back resolvers?

**Comment by ladar** [comment]
I've tried using this, and it caused a number of the box creation configs to fail, particularly on Hyper-V. Setting a `FALLBACK` also caused problems on the boxes I tried it with.

I'm still traveling, so part of the problem could have been the crappy hotel DNS servers. I plan to play with this more when I get a Windows built bot setup at the datacenter.

I'm also hoping that when I finally get the robox blade servers online it will be easier to test out changes like this across all the different box/distro/platform combinations. 

Long story short, this request isn't dead, just pending further research.

**Comment by Pesa** [comment]
@ladar any updates?
